src/view/reviewManager.ts

```diff
@@ -1025,6 +1028,55 @@ export class ReviewManager implements vscode.DecorationProvider {
 		await this.validateState();
 	}
 
+	public async createPullRequest(): Promise<void> {
+		const branchName = this._repository.state.HEAD.name;
```

Comment:
right now if you try to create a PR from local branch that hasn't been pushed to the remote, a notification with just "Validation Failed" is shown. should we push the branch in that case?

I think there may also be a problem when the local branch is named differently than the remote branch

Reply:
for the first iteration, we can show a warning for that case. We didn't decide yet how we are going to handle the `push` workflow.